source/gent/tier1_hortense.rst

```diff
@@ -151,6 +176,28 @@ Project storage
   cd $VSC_SCRATCH_PROJECTS_BASE/<yourprojectnamehere>
 
 
+Read only filesystem
+********************
+
+The performance of the filee system is not always optimal in case of small files often read without
+writing them (software stack, certain input files, for example ones for DNA sequencing). 
+For those files we defined a read-only partition (mirroring the whole network file system). 
```

Comment:
`For this use case, we provide a read-only version of the scratch` (i don't like network file system, it's only for scratch, not for all such file systems like the nfs automounts)